components/menu/stories/template.js

```diff
@@ -217,6 +219,7 @@ export const MenuItem = (
 
 export const MenuGroup = (
 	{
+		hasActions = false,
```

Comment:
Does this need to be in the `argTypes` at all? I saw it in [the `spectrum-two` menu](https://github.com/adobe/spectrum-css/blob/10392df161f93293cf04b38f2d49009c3fb6d998/components/menu/stories/menu.stories.js#L45), so it might not need to move into this branch. (or it's just an arg for the MenuItem)

Reply:
Yeah, it's just an `argType` for `MenuItem`. Menu is weird because it has states/args that apply to the full menu, but then also ones for the individual items, and in a lot of cases there's not a lot of overlap.